assets/src/design-system/components/dropDown/index.js

```diff
@@ -63,7 +62,7 @@ export const DropDown = ({
   onMenuItemClick,
   options = [],
   placement = PLACEMENT.BOTTOM,
-  popupFillWidth = DEFAULT_POPUP_FILL_WIDTH,
```

Comment:
Nice catch here, I copied and pasted and was so concerned about maintaining use cases I didn't catch that this is a bool.